src/lib.rs

```diff
@@ -584,15 +590,86 @@ pub fn init_and_run(options: Options) -> ! {
 
                             project_status.prevent_overwrite_status = None;
                         }
-                        SaveModalResult::Nothing => {}
+                        SaveModalResult::Nothing => (),
                     }
                 }
 
-                if reset_viewport {
+                if input_state.camera_reset_viewport || menu_status.reset_viewport {
                     camera_interpolation =
                         Some(CameraInterpolation::new(&camera, &scene_bounding_box, time));
                 }
 
+                if menu_status.export_obj {
+                    if let Some(path) = tinyfiledialogs::save_file_dialog_with_filter(
+                        "Save",
+                        "export.obj",
```

Comment:
This is a good filename when the current project hasn't been saved yet. The export usually suggests the same filename as the current file, just a different extension, so of the project is called `my_model.hurban`, the the suggested export file name should be `my_model.obj`.